plasma_framework/docs/integration-docs/integration-doc.md

```diff
@@ -64,11 +64,20 @@ To determine the position of the outputs of a transaction, you will include the
 
 
 
-## Transaction type and output type
+## Transaction type, output type, and spending condition
 
-ALD introduces the concepts of transaction type, and transcation output type. Each transaction type and transcation output type can define different rules about how to spend funds.
+ALD introduces the concepts of transaction type and transaction output type.
+Each transaction output type can define its own rule about how funds may be spent.
 
+Rules are encoded in the [mapping](../contracts/src/exits/registries/SpendingConditionRegistry.sol) from `hash(tx_type, output_type)` to an address.
+If address is 0x0, spend is forbidden.
+Otherwise, address is referencing a contract that implements [ISpendingCondition](../contracts/src/exits/interfaces/ISpendingCondition.sol) interface and its `verify(...) returns (bool)` function.
 
+For a valid transaction, `verify(...)` will return `true` for every of transaction's inputs.
+
+## Printing money
```

Comment:
I'd opt for a less jargon-ey title :thinking: "Correct transaction balancing", "Correct Input/Output balance", "Transaction balance validation"?

And also in general, so that it is "positively phrased", so "Avoiding printing money" at the minimum.

Reply:
`Transaction balance validation` it is.